app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
```

Comment:
Using `keys()` is redundantly here, just `"vaccine" not in vstr` is enough.
Use `vstr["vaccine"] is None` instead of using `==`.
Also, this condition can be simpler, just `if not vstr.get(vaccine):`, it will be equal